library/VTKExtensions/Rendering/vtkF3DRenderer.h

```diff
@@ -173,6 +175,14 @@ class vtkF3DRenderer : public vtkOpenGLRenderer
   void ConfigureHDRISkybox();
   ///@}
 
+  ///@{
+  /**
+   * Methods to check if certain HDRI caches are available
```

Comment:
imo you can specify which hdri caches could be considered ?

Reply:
you mean in the comment ? No need, no doxygen doc is being built for this class.